src/riscv-integration.adoc

```diff
@@ -1101,6 +1103,10 @@ NOTE: `auth_cap` is <<ddc>> for Legacy mode and `cs1` for Capability Mode
 
 ^1^ This check is architecturally required, but is impossible to encounter so may not required in an implementation.
 
+^2^ Accesses that are not naturally aligned can wrap around the address space
+from 2^XLEN^ to 0. A capability's bounds include both 0 and 2^XLEN^-1 if and
+only if they are the same as the bounds of <<infinite-cap>>.
```

Comment:
I think there is an edge case when XLENMAX=64 but the effective XLEN=32 because you can have a capability (that is not Infinite) whose bounds cover the first (2^32)-1 and memory accesses wrap at 2^32 as required by RISC-V.

Reply:
I think the RISC-V spec is ambiguous here which Allen Baum tried to clear up on the mailing list: https://lists.riscv.org/g/tech-privileged/message/1116. The conclusion seems to be that setting out of range bits will cause an access fault,and so wrapping behaviour is not expected.